Sources/OTelSemanticConventions/Generated/AttributeNames/OTelAttribute+http.swift

```diff
@@ -37,11 +37,7 @@ extension OTelAttribute {
             ///
             /// - Type: templateStringArray
             ///
-            /// Instrumentations SHOULD require an explicit configuration of which headers are to be captured. Including all request headers can be a security risk - explicit configuration helps avoid leaking sensitive information. The `User-Agent` header is already captured in the `user_agent.original` attribute. Users MAY explicitly configure instrumentations to capture them even though it is not recommended. The attribute value MUST consist of either multiple header values as an array of strings or a single-item array containing a possibly comma-concatenated string, depending on the way the HTTP library provides access to headers.
-            ///
-            /// - Examples:
-            ///     - `http.request.header.content-type=["application/json"]`
-            ///     - `http.request.header.x-forwarded-for=["1.2.3.4", "1.2.3.5"]`
+            /// Instrumentations SHOULD require an explicit configuration of which headers are to be captured. Including all request headers can be a security risk - explicit configuration helps avoid leaking sensitive information.  The `User-Agent` header is already captured in the `user_agent.original` attribute. Users MAY explicitly configure instrumentations to capture them even though it is not recommended.  The attribute value MUST consist of either multiple header values as an array of strings or a single-item array containing a possibly comma-concatenated string, depending on the way the HTTP library provides access to headers.  Examples:  - A header `Content-Type: application/json` SHOULD be recorded as the `http.request.header.content-type`   attribute with value `["application/json"]`. - A header `X-Forwarded-For: 1.2.3.4, 1.2.3.5` SHOULD be recorded as the `http.request.header.x-forwarded-for`   attribute with value `["1.2.3.4", "1.2.3.5"]` or `["1.2.3.4, 1.2.3.5"]` depending on the HTTP library.
```

Comment:
Did the formatting of this documentation change in the source model yamls? That's unfortunate if so, as we've lost our nice list format.

Reply:
Oh it looks like they moved the examples from [a separate `examples` field in v1.29](https://github.com/open-telemetry/semantic-conventions/blob/v1.29.0/model/http/registry.yaml#L20) to [inside of the `note` field in v1.34](https://github.com/open-telemetry/semantic-conventions/blob/v1.34.0/model/http/registry.yaml#L20).

No concerns for this pull request. We probably ought to adjust the Generator to respect the newlines in the `note` field though.

Reply:
Good catch! I created an issue for this too.